spread.yaml

```diff
@@ -414,8 +414,15 @@ debug-each: |
 
         echo '# journal messages for snapd'
         get_journalctl_log -u snapd
-        echo '# apparmor denials '
-        dmesg --ctime | grep DENIED || true
+        case "$SPREAD_SYSTEM" in
+            fedora-*|centos-*|amazon-*)
+                ausearch -m AVC
+                ;;
+            *)
+               echo '# apparmor denials '
+               dmesg --ctime | grep DENIED || true
+               ;;
+        esac
```

Comment:
+1

Reply:
Do you think we are a ready for a spread test that only passes if one can snap install hello on fedora with 0 warnings?

Reply:
Yes, that's part of my branch.